rel-eng/tito.props

```diff
@@ -529,3 +529,8 @@ whitelist = katello
   rubygem-runcible
   rubygem-typhoeus
   rubygem-zest
+
+[katello-nightly-pulpcore-el7]
+disttag = .el7
+whitelist =
+        python-setuptools-scm
```

Comment:
Nit, indented more than other whitelists in the file.

Reply:
blame the tools ;)
but fair